Autopilot/Inc/airspeed.hpp

```diff
@@ -54,16 +50,11 @@ class airspeed {
          *  ensure that data acquired makes sense, has been
          *  gathered recently within reason (past 10s?)
          * */
-        virtual void GetResult(airspeedData_t &Data) = 0; 
+        virtual void GetResult(airspeedData_t *Data) = 0; 
 };
 
 class dummyairspeed: public airspeed{
```

Comment:
Go ahead and delete the entire dummy airspeed class, its only reason for existing was to make other shit work while the real class was developed